src/components/WeekDayCard.css

```diff
@@ -0,0 +1,115 @@
+.WeekDayCard {
+  width: 95vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  color: var(--color-eight);
+}
+
+.weekday-headline {
+  margin: 0 0 10px 10px;
+  font-size: 1.2rem;
+  text-shadow: 2px 2px 2px rgba(150, 150, 150, 1);
+}
+
+.weekday-box {
+  height: auto;
+  width: 95vw;
+  background-color: var(--color-four-opaque-dark);
+  backdrop-filter: blur(5px);
+  border-radius: 30px;
+  border: 1px solid var(--color-four-opaque);
+  margin-bottom: 40px;
+  position: relative;
+  box-shadow: 0 15px 30px 5px rgba(0, 0, 0, 0.3);
+  display: grid;
+  grid-template-columns: [left] 47.5vw [right] 47.5vw;
+  grid-template-rows: 20vw auto 20vw;
+}
+
+.weekday-box::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 50%;
+  background-color: rgba(255, 255, 255, 0.287);
+  border-radius: 30px 0 0 30px;
+}
+
+.weekday-box-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  grid-column-start: 1;
+  grid-column-end: 3;
+  padding: 5px 25px 0;
+  color: var(--color-ten);
+  font-size: 1.5rem;
+}
+
+.sun {
+  font-size: 2.1rem;
+}
+
+.product-name-list {
+  list-style: none;
+  padding-left: 15px;
+}
+
+.product-name-right {
+  margin-bottom: 10px;
+  font-size: 0.8rem;
+  background-color: var(--color-eleven);
+  border-radius: 5px;
+  text-align: end;
+  margin-right: 15px;
+  padding: 3px;
+  box-shadow: inset 4px 4px 8px #bbadade1, inset -4px -4px 8px #fdebeb;
+}
+
+.product-name-left {
+  margin-bottom: 10px;
+  font-size: 0.8rem;
+  background-color: var(--color-twelve);
+  border-radius: 5px;
+  padding: 3px;
+  box-shadow: inset 4px 4px 8px #9d8585, inset -4px -4px 8px #d5b3b5;
+}
+
+.morning {
+  grid-column-start: 1;
+  grid-column-end: 2;
+  grid-row-start: 2;
+  grid-row-end: 3;
+}
+
+.evening {
+  grid-column-start: 2;
+  grid-column-end: 3;
+  grid-row-start: 2;
+  grid-row-end: 3;
+}
```

Comment:
You made it work to appear on the right side!!!!! Amazing job!! It just looks fantastic 🎉